app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt

```diff
@@ -38,20 +39,20 @@ object JavaScriptActionHandler {
     }
 
     @JvmStatic
-    fun setUp(topMargin: Int): String {
+    fun setUp(): String {
         val app: WikipediaApp = WikipediaApp.getInstance()
-        return String.format("pcs.c1.Page.setup({" +
-                "platform: pcs.c1.Platforms.ANDROID," +
-                "clientVersion: '%s'," +
-                "theme: pcs.c1.Themes.%s," +
-                "dimImages: %b," +
-                "margins: { top: '%dpx', right: '%dpx', bottom: '%dpx', left: '%dpx' }," +
-                "areTablesInitiallyExpanded: %b," +
-                "textSizeAdjustmentPercentage: '100%%'," +
-                "loadImages: %b" +
-                "})", BuildConfig.VERSION_NAME, app.currentTheme.funnelName.toUpperCase(),
+        return String.format("{" +
+                "\"platform\": \"pcs.c1.Platforms.ANDROID\"," +
+                "\"clientVersion\": \"%s\"," +
+                "\"theme\": \"%s\"," +
+                "\"dimImages\": \"%b\"," +
```

Comment:
Boolean parameters do not need quotes around them. In fact, this causes them to break.